operator/pkg/reconciliation/construct_service.go

```diff
@@ -65,9 +65,24 @@ func newServiceForCassandraDatacenter(dc *api.CassandraDatacenter) *corev1.Servi
 
 	utils.AddHashAnnotation(service)
 
+	addAdditionalOptions(service, &dc.Spec.AdditionalServiceConfig.DatacenterService)
```

Comment:
I think we should put this above the `AddHashAnnotation` call, or it won't roll out an update to the service

Reply:
I totally missed that. Thanks 😄